src/battle_controller_link_opponent.c

```diff
@@ -463,7 +463,7 @@ static void LinkOpponentHandleDrawTrainerPic(u32 battler)
     }
 
     BtlController_HandleDrawTrainerPic(battler, trainerPicId, TRUE,
-                                       xPos, 40 + 4 * (8 - gTrainerSprites[trainerPicId].y_offset),
+                                       xPos, 40,
                                        -1);
```

Comment:
This all can probably go on the same line no since you removed the biggest part. Same for all the other instances